### PR TITLE
Deployment Procedure for trusted-bigdl-llm-serving on Kubernetes

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Deployment Procedure for trusted-bigdl-llm-serving on Kubernetes

### CODE REVIEW

**Comment by glorysdj** [comment]
should also remove /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx

File: ppml/tdx/docker/trusted-bigdl-llm/serving/docker/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Build bigdl-trusted-bigdl-llm-fschat tdx image
+```bash
+docker build \
+  --build-arg http_proxy=.. \
+  --build-arg https_proxy=.. \
+  --build-arg no_proxy=.. \
+  --rm --no-cache -t intelanalytics/bigdl-ppml-trusted-bigdl-llm-serving-tdx:2.4.0-SNAPSHOT .
+```
```
Comment (glorysdj) [inline ppml/tdx/docker/trusted-bigdl-llm/serving/docker/README.md:8]: please add more readme for how to run the trusted llm serving on TDX

**Comment by Romanticoseu** [comment]
> should also remove /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx

I have move /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx to /ppml/tdx/docker/trusted-bigdl-llm/serving/

**Comment by glorysdj** [comment]
> > should also remove /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx
> 
> I have move /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx to /ppml/tdx/docker/trusted-bigdl-llm/serving/

https://github.com/intel-analytics/BigDL/tree/538ca44f1f92d23a3eb74698a141f751485f5b96/ppml/tdx/docker/trusted-bigdl-llm-serving-tdx is duplicated, need to be deleted.

**Comment by Romanticoseu** [comment]
> > > should also remove /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx
> > 
> > 
> > I have move /ppml/tdx/docker/trusted-bigdl-llm-serving-tdx to /ppml/tdx/docker/trusted-bigdl-llm/serving/
> 
> https://github.com/intel-analytics/BigDL/tree/538ca44f1f92d23a3eb74698a141f751485f5b96/ppml/tdx/docker/trusted-bigdl-llm-serving-tdx is duplicated, need to be deleted.

Ok, I have resolved this issue.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**